### PR TITLE
Status view

### PR DESCRIPTION
**Pull request by temp666**
The dialog displays the last position update.
Helpful to detect if a phone has crashed
grafik

### CODE REVIEW

File: madmin/templates/status.html [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+            "columns": [
+                { data: 'name', title: 'Name' },
+                { data: 'lastUpdate', title: 'Lastupdate' },
+                { data: 'lat', title: 'Position' }
+            ],
+            "columnDefs": [
+                {
+                    "targets": [1],
+                    "render": function (data, type, row) {
+                        return moment.unix(data).format('YYYY-MM-DD HH:mm:ss');
```
Comment (simonbuehler) [inline madmin/templates/status.html]: maybe prefix with a "(x minutes ago)" for better readability,

**Comment by georgeherby** [comment]
Like the idea. Guess no help if RGC is running and the code is still moving the walker but injection has died and not actually reporting anything?

File: madmin/templates/status.html (hunk 1 above)
Comment (georgeherby) [reply madmin/templates/status.html]: Agree on this

**Comment by muckelba** [comment]
Maybe there's a way to extract the timestamp of the most recent data of each phone. I know that mad is already logging this

**Comment by AboveColin** [comment]
Great addition 👍

**Comment by temp666** [comment]
this dialog is reworked with more information.
status2

**Comment by cecpk** [comment]
yes! 💃

**Comment by temp666** [comment]
fix problem with iv_mitm and database sturcture.
please delete the table trs_status, if you have already tested this PR

**Comment by cecpk** [comment]
@temp666 pls rework for branch `mitmbase`
thx

**Comment by JimBim89** [comment]
init mode and reboot option always seem to display "true".

**Comment by temp666** [comment]
PR is ready for ```mitmbase```

**Pull request merged**